src/index.ts

```diff
@@ -1141,6 +1141,14 @@ export function isLocationEnabledSync() {
   return false;
 }
 
+export function isHeadphonesConnected() {
+  return RNDeviceInfo.isHeadphonesConnected();
```

Comment:
These need to be restricted to android and ios
```suggestion
  if (OS === 'android' || OS === 'ios') {
    return RNDeviceInfo.isHeadphonesConnected();
  }
  return false;
```